src/app/application/components/app-collection/AppCollection.vue

```diff
@@ -51,7 +51,7 @@ type ChangeValue = {
 }
 
 const props = defineProps<{
-  total: number,
+  total: number | undefined,
```

Comment:
**Comment**: It’s better to define props using an object literal because then you can actually define this default and don’t have to handle it in code again.

The type parameter style of defining props doesn’t allow for default values to be defined (unless you use that cumbersome `withDefaults` helper https://vuejs.org/api/sfc-script-setup.html#defineprops-defineemits)

Reply:
this is done